todo_app/data/session_items.py

```diff
@@ -1,72 +1,150 @@
 from flask import session
+import requests
+import json
 
+import os
 
+boardid = os.environ.get('boardid')
+trello_key = os.environ.get('trello_key')
+trello_token = os.environ.get('trello_token')
 
-_DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
-]
 
+listtodo = os.environ.get('listtodo')
+listdoing = os.environ.get('listdoing')
+listdone = os.environ.get('listdone')
 
 
 def get_items():
-    """
-    Fetches all saved items from the session.
 
-    Returns:
-        list: The list of saved items.
     """
-    return session.get('items', _DEFAULT_ITEMS.copy())
-
-
-def get_item(id):
+    Module 2 - amend get_items to use response and Trello api
+    Returns: response json() containing trello card names / id's and lists id's
     """
-    Fetches the saved item with the specified ID.
 
-    Args:
-        id: The ID of the item.
+    url = 'https://api.trello.com/1/boards/' + boardid +'/cards?fields=name,idList'
 
-    Returns:
-        item: The saved item, or None if no items match the specified ID.
-    """
-    items = get_items()
-    return next((item for item in items if item['id'] == int(id)), None)
+    query = {
+        'key': trello_key,
+        'token': trello_token
+            }
+
+    response = requests.request(
+            "GET",
+            url,
+            params=query
+            )
 
+    return make_get_request(url, query)
```

Comment:
Looks like you make the request twice!

Reply:
resolved
this has been done , will be visible with next PR